plugins/chat/spec/lib/chat/onebox_handler_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Chat::OneboxHandler do
```

Comment:
Something raises questions here. We describe `Chat::OneboxHandler` but mostly the spec is calling `InlineOneboxer` and `Oneboxer.preview`, what are we testing in reality?